src/lib/config-schema.json

```diff
@@ -79,11 +83,27 @@
     },
     "theme": {
       "type": "object",
-      "required": ["compensation", "lines", "skinparams", "states", "compositeStates"],
+      "required": ["compensation", "lines", "skinparams", "states", "compositeStates", "comments"],
       "properties": {
         "compensation": {
           "$ref": "#/$defs/compensation"
         },
+        "compositeStates": {
```

Comment:
re-ordered the properties to be alphabetical so it matches the previous types. This was useful to verify the generated types were the same as the previous types.